main.go

```diff
@@ -1,15 +1,40 @@
 package main
 
 import (
+	"os"
+	"time"
+
 	"github.com/RedHatInsights/entitlements-api-go/config"
 	"github.com/RedHatInsights/entitlements-api-go/controllers"
 	"github.com/RedHatInsights/entitlements-api-go/logger"
 	"github.com/RedHatInsights/entitlements-api-go/server"
 
 	"github.com/sirupsen/logrus"
+
+	"github.com/getsentry/sentry-go"
 )
 
 func main() {
+
+	var dsn string = os.Getenv("SENTRY_DSN")
+
+	if dsn != "" {
+		err := sentry.Init(sentry.ClientOptions{
+			// Either set your DSN here or set the SENTRY_DSN environment variable.
+			//Dsn: "https://examplePublicKey@o0.ingest.sentry.io/0",
+			// Either set environment and release here or set the SENTRY_ENVIRONMENT
+			// and SENTRY_RELEASE environment variables.
+			//Environment: "",
+			//Release:     "my-project-name@1.0.0",
+			// Enable printing of SDK debug messages.
+			// Useful when getting started or trying to figure something out.
+			Debug: false,
+		})
```

Comment:
I think we could simplify this to just be:
```suggestion
		err := sentry.Init(sentry.ClientOptions{})
```
since the `SENTRY_DSN` will be picked up in an env var (assuming that's what we'll set) and we can leave the rest out?